src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(0) {
```

```suggestion
.stars--0 :nth-child(0),
.stars--1 :nth-child(-n + 1),
... {
```
Consider grouping your selectors with commas